pkg/controllers/deprovisioning/multimachineconsolidation.go

```diff
@@ -66,7 +67,8 @@ func (m *MultiMachineConsolidation) ComputeCommand(ctx context.Context, candidat
 	}
 
 	if !isValid {
-		return Command{}, fmt.Errorf("command is no longer valid, %s", cmd)
+		logging.FromContext(ctx).Debugf("consolidation command is no longer valid, %s", cmd)
```

Comment:
nit: thoughts on making this more specific?
```suggestion
		logging.FromContext(ctx).Debugf("multi-node consolidation command is no longer valid, %s", cmd)
```

Reply:
going to leave for now since the output is pretty clear when the candidate are listed for the action.